packages/grafana-ui/package.json

```
@@ -69,6 +69,7 @@
     "@storybook/addon-docs": "5.3.9",
     "@storybook/addon-info": "5.3.9",
     "@storybook/addon-knobs": "5.3.9",
+    "@storybook/addon-storysource": "^5.3.14",
```

```suggestion
    "@storybook/addon-storysource": "5.3.14",
```
Lock the version. You'll probably have to run `yarn add @storybook/addon-storysource@5.3.14` otherwise the CI will complain.

---

Good to know! I see there are a few packages with non-locked dependencies, so wondering if those should be changed as well.